lib/models.py

```diff
@@ -142,7 +142,7 @@ def import_gobject_from_dashd(self, dashd, rec):
         try:
             newdikt = subdikt.copy()
             newdikt['object_hash'] = object_hash
-            if subclass(**newdikt).is_valid() is False:
+            if subclass(**newdikt).is_valid(dashd) is False:
```

Comment:
The classes here (`subclass`) are `Proposal` and `Superblock`, which don't accept any arguments to `is_valid`, so this change doesn't make sense and also isn't valid.